packages/top-app-bar/index.js

```diff
@@ -69,10 +69,26 @@ export default class TopAppBar extends React.Component {
     this.foundation_.destroy();
   }
 
+  componentDidUpdate(prevProps) {
+    const foundationChanged = ['short', 'shortCollapsed', 'fixed']
+      .filter((variant) => this.props[variant] !== prevProps[variant] )
```

Comment:
I think some is more appropriate here 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Awesome, I was not familiar with some. Shall I make the change?

Reply:
(I made the change)

Reply:
lol thanks! .some will test each one until one returns true. Kind of like .every